docs/src/navigation/breadcrumbs/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import {
+    TitleBar,
+    Typography,
+} from 'react-cm-ui';
+import Main from '../../global/main';
+import MarkdownContainer from '../../global/markdownContainer';
+import Heading from '../../global/heading';
+import BreadcrumbsSample from './breadcrumbsSample';
+import BreadcrumbsSampleAllRoutes from './breadcrumbsSampleOnlyPreviousRoute';
+import BreadcrumbsSampleSeparatorIcon from './breadcrumbsSampleSeparatorIcon';
+import ComponentApi from '../../global/componentApi';
+import Example from '../../global/example';
+// eslint-disable-next-line import/no-named-default, import/extensions
+import { default as breadcrumbsDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/navigation/breadcrumbs/breadcrumbs';
+
+function BreadcrumbsExample() {
+    return (
+        <Main page="headers">
+            <TitleBar title="Breadcrumbs" />
```

Comment:
Please change this too.

```jsx
function DocsBreadcrumbs() {
    const {
        description,
        name,
    } = rootDoc;

    return (
        <Main page={camelCase(name)}>
            <TitleBar title={startCase(name)} />
            
            ...
        </Main
```

Reply:
NOTE: It is likely `displayName` instead of `name` (see [discussion](https://github.com/saddlebackdev/react-cm-ui/pull/194#discussion_r477571052) on #194).

Reply:
You are right @groberts314, thank you for the catch.